src/components/learner-credit-management/data/constants.js

```diff
@@ -34,6 +34,12 @@ export const CONTENT_TYPE_COURSE = 'course';
 export const EXEC_ED_TITLE = 'Executive Education';
 export const EXEC_COURSE_TYPE = 'executive-education-2u';
```

Comment:
pre-existing/unrelated nit: may want to rename this to `EXEC_ED_COURSE_TYPE` while we're in here

Reply:
Done!